packages/serve/src/types.ts

```diff
@@ -43,6 +43,7 @@ export type devServerOptionsType = {
     | false
     | string
     | transportModeEnum
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
This was just an oustanding warning in CI

Reply:
We can accept this change, if you can revert the others.

Reply:
Split this out into https://github.com/webpack/webpack-cli/pull/3048, if you want to leave this open till you drop 10, or just close this one since it will get outdated till you're read to drop 10